new_dawn_server/actions/api/resources.py

```diff
@@ -58,13 +59,16 @@ def create_match(user_from_id, user_to_id):
             entity_id=0,
             entity_type=EntityType.NONE.value
         ).save()
+        # NotificationService.send_notification([str(user_from_id), str(user_to_id)], message="You are matched")
 
     def obj_create(self, bundle, **kwargs):
         super(UserActionResource, self).obj_create(bundle).obj.save()
-        if UserAction.objects.filter(user_to_id=bundle.data.get("user_from_id"),
-                                     user_from_id=bundle.data.get("user_to_id"),
-                                     action_type=ActionType.LIKE.value).exists():
-            self.create_match(bundle.data.get("user_from_id"), bundle.data.get("user_to_id"))
+        if bundle.data.get("action_type") == ActionType.LIKE.value:
+            # NotificationService.send_notification([str(bundle.data.get("user_to_id"))], message="You are liked")
```

Comment:
Why comment it out?

Reply:
Otherwise it will send notification to user-id "1" everytime people get it gets matched or liked, which happens a lot in unit-test.

Reply:
In that case you should use “patch” to mock this function in unit test, so it won’t really send the notification.

You can learn how to do that in phone verification test.

Reply:
This function will send data to beams server, and that server will send to Apple notification service. 
If I un-comment it, the "like" and "match" action in our unit tests will send notification to associated user-ids like "1" or "2" or "3".

Reply:
Yeah but if you mock the send_notofication function in unit test, the function will become fake and wouldn’t send anything out

Reply:
It was not because I created a test for this function though, it was because this function lives inside the "like" and "match" logic.

Reply:
Yeah I know. What I mean is you should mock this function in the “like” or “match” test so whenever this function get called in like/match unit test, this function can be auto-replace with a fake one.

Reply:
You might want to take a look at how phone-verication is mocked in other tests